scripts/find-unused-images.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+""" This script identifies images that are not used in any doc pages
+so they can be removed. Run it from within the scripts directory. """
+
+images = list()
+for root, dirs, files in os.walk("../images"):
+    for f in files:
+        images.append(f)
+print("Total # of image files:", len(images))
+
+dirs = list()
+for path, subdirs, files in os.walk("../"):
+    dirs.append(path)
+
+docs = list()
+for d in dirs:
+    for path, subdirs, files in os.walk(d):
+        for name in files:
+            docs.append(os.path.join(path, name))
+
+unused_images = images
+unread_files = list()
+for doc in docs:
```

Comment:
are you missing a commit? The change we added to skip the `not_checked_for_images.txt` file isn't there. Ditto the `onError` argument for `os.walk`

Reply:
Sorry. Added `onerror` back in. I'm no longer writing to `not_checked_for_images.txt`, so skipping isn't necessary.